src/runtime12.js

```diff
@@ -93,10 +93,7 @@ function requestAnimationFrame(fn) {
 }
 
 function __runRAFHandlers() {
-    var handlers_copy = [];
-    for (var i = 0; i < RAF_LISTENERS.length; i++) {
-        handlers_copy.push(RAF_LISTENERS[i]);
-    }
+    var handlers_copy = RAF_LISTENERS;
```

Comment:
why this change?

Reply:
Turns out the book had diverged from the source code and it wasn't being caught before.

Reply:
This moves the book code into the code code. I think the book code is nicer.

Reply:
This new code is a pointer copy instead of a deep copy. Don't we need a deep copy?

Reply:
Ah great question. It's not a copy at all—it's a move, on the next line we reset `RAF_LISTENERS` to a newly-allocated vector. So what's important here is that `RAF_LISTENERS` and `handlers_copy` are disjoint, and the move is more convenient (and I guess faster, not that we care) than the deep copy & clear.